native-sql-engine/cpp/src/precompile/wscgapi.hpp

```diff
@@ -3,8 +3,10 @@
 #include <arrow/buffer.h>
 #include <arrow/compute/api.h>
 #include <arrow/record_batch.h>
+#include <arrow/util/value_parsing.h>
```

Comment:
`#include <arrow/vendored/fast_float/parse_number.h>` should be added here.
And also this arrow pr should be added before this pr merged. https://github.com/oap-project/arrow/pull/173/files